Lexplorer/Components/HomepageOverview.razor

```diff
@@ -154,13 +176,15 @@
     private Transactions? transactionData;
     private List<Lexplorer.Models.Pair>? pairs { get; set; } = new();
     private Dictionary<string, UniswapToken> uniswapTokens = new();
+    private LoopStatsDailyCount dailyCount { get; set; } = new();
 
     private double averageBlockTime;
     private long averageTransactions;
     private double lastBlockSubmittedTime;
 
     protected override async Task OnInitializedAsync()
     {
+        dailyCount = await LoopStatsService.GetDailyCount();
```

Comment:
That would be great, I will implement it :)

Reply:
As to why it was the very first, that was just a judgement call I made because this is a small and in-expensive call. Can be put in another order if wanted.

Reply:
Have now added 10 minute cache.